### DISCUSSION TITLE
libcrypto.so.3 available version issue

### DISCUSSION THREAD

**Discussion by maroof-siddiqui-aurion** [community/community#146717] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

The libcrypto.so.3 is currently is on 3.0.15 and we wanted to upgrade it to 3.0.16 to close venerability points

Path             : /opt/oracle.ahf/python/lib/libcrypto.so.3
Reported version : 3.0.15
Fixed version    : 3.0.16

recommended action is to Upgrade to OpenSSL version 3.0.16 or later.

But we are unable to find the above version..

**Comment by queenofcorgis** [comment, marked as answer]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

Please note: The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
